rs/messaging/src/routing/stream_handler/tests.rs

```diff
@@ -2117,18 +1922,11 @@ fn induct_stream_slices_with_messages_to_migrating_canister() {
             );
 
             assert_eq!(expected_state, inducted_state);
-
-            // One reject response enqueued.
             assert_eq!(
-                initial_available_guaranteed_response_memory - reject_response_count_bytes as i64,
+                stream_handler.available_guaranteed_response_memory(&inducted_state),
                 available_guaranteed_response_memory
             );
```

Comment:
E.g. In this particular case, I believe the earlier assert(s) were more useful than the current one.

Actually, how come we went from `assert!(a >= b)` (which, at least according to the comment was actually always `<`) to `assert_eq!(a, b)`?

Same below.

Reply:
It is now equal because the reject response is never in the stream because it is generated locally from the reject signal and then inducted immediately.